test/transformer/descriptor/create-mock-list-values.test.ts

```diff
@@ -1,41 +0,0 @@
-import { createMockList } from 'ts-auto-mock';
```

Comment:
Didn’t you remove these tests in a different commit?

Reply:
I don't think see. the duplicated test is inside test/transformer/create-mock-list-values.test.ts and create-mock-list.test.ts